packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/GenerateSummaryHandler.java

```diff
@@ -0,0 +1,16 @@
+package software.uncharted.terarium.hmiserver.service.tasks;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+// @Slf4j
+public class GenerateSummaryHandler extends TaskResponseHandler {
```

Comment:
Just a reminder here that we will want to have a discussion about the handler logic here as we will want to track the results coming back in Postgres table (e.g. uniqueId => summary_text).